kubernetes/base/load-balancer.yml

```diff
@@ -0,0 +1,22 @@
+kind: Service
+apiVersion: v1
```

Comment:
In other files is `apiVersion` before `kind`. It's nice to keep the structure of files unified.
Same apply for all files in service directory